test/lib/processors/themeBuilder.js

```diff
@@ -29,7 +29,7 @@ function prepareResources({library} = {}) {
 		string: input
 	});
 
-	memoryAdapter.write(resource);
+	await memoryAdapter.write(resource);
```

Comment:
I guess this problem now became visible after https://github.com/SAP/ui5-fs/pull/407 which adds another "Promise" (although being sync most of the time).

This will probably also reveal issues in other code such as custom tasks, which is good, as this can always lead to issues.
But I'm also thinking about the overhead and impact that the change brings as it creates another promise instance on every resource write.

Reply:
> But I'm also thinking about the overhead and impact that the change brings as it creates another promise instance on every resource write.

Yes, that was also my concern